.gitignore

```diff
@@ -1,4 +1,5 @@
 /node_modules/
 /package-lock.json
 /dist/api
-/dist/main.js
+/dist/main.*
+/dist/sources.*
```

Comment:
Why not just `/dist/`?

Reply:
Because `dist/index.html` is (and was already before) part of the git source, it's the example entry point. (ideally it should not be in dist IMHO, I can change that in a follow-up PR if you agree)

Reply:
+1 :-)